Formula/git-tools.rb

```diff
@@ -0,0 +1,39 @@
+class GitTools < Formula
+  include Language::Python::Shebang
+
+  desc "Assorted git-related scripts and tools"
+  homepage "https://github.com/MestreLion/git-tools"
+  url "https://github.com/MestreLion/git-tools/archive/v2022.12.tar.gz"
+  sha256 "b01e3b8de268ee07854e6cc66d78db6ed6cbc3e71dfeded8456138881bdf97f1"
+  license "GPL-3.0-only"
+  head "https://github.com/MestreLion/git-tools.git", branch: "main"
+
+  depends_on "python@3.11"
+  uses_from_macos "bash"
+
+  def install
+    rewrite_shebang detected_python_shebang, "git-restore-mtime"
+    bin.install Dir["git-*"]
+    man1.install Dir["man1/*.1"]
```

Comment:
```suggestion
    bin.install testpath.glob("git-*")
    man1.install testpath.glob("man1/*.1")
```
The effect is equivalent, but `Dir` makes it seem like you're operating over directories when you're not.

Reply:
Not sure if `testpath.glob` can be used outside `test do`:
```
Error: An exception occurred within a child process:
  NoMethodError: undefined method `glob' for nil:NilClass
```

From what I can see a lot of formulae are using `Dir` e.g.:
```
putmail-queue.rb:    man1.install Dir["man/man1/*.1"]
docker-buildx.rb:    doc.install Dir["docs/reference/*.md"]
druid.rb:    bin.install Dir["#{libexec}/bin/*.sh"]
```

Reply:
Sorry, I meant `buildpath` instead of `testpath`.